### PR TITLE
docs: in-progress changelog for v8.0

### PR DESCRIPTION
**Pull request by brendankenny**
see #11866

to merge into #12593

### CODE REVIEW

**Review by connorjclark — APPROVED** [review]

File: changelog.md [hunk 1]
```diff
@@ -1,71 +1,75 @@
...
-## Notable Changes
-~~ TODO: Move notable changes here ~~
+## Notable changes
+* The **Performance Category** had a number of scoring changes to align with other performance tools and to better reflect the state of the web:
+  - TODO: decide on FAQ incorporation and either replace these with that or flesh these out
+  - scoring: rebalance perf metric weightings ([#12577](https://github.com/GoogleChrome/lighthouse/pull/12577))
+  - scoring: update TBT score curve ([#12576](https://github.com/GoogleChrome/lighthouse/pull/12576))
+  - update cumulative-layout-shift ([#12554](https://github.com/GoogleChrome/lighthouse/pull/12554))
+  - update FCP score curve ([#12556](https://github.com/GoogleChrome/lighthouse/pull/12556))
+* The report includes a new metric filter. Pick a metric to focus on the opportunities and diagnostics most relevant to improving just that metric:
```
Comment (addyosmani) [inline changelog.md:15]: Is it worth linking up to the score calc + include a screenshot of it for the perf category section?

Comment (paulirish) [reply changelog.md:15]: for fun i added that in https://github.com/GoogleChrome/lighthouse/pull/12597

Comment (addyosmani) [reply changelog.md:15]: looks good!

**Comment by paulirish** [comment]
@googlebot I consent

**Pull request merged**